docs/PID tuning.md

```diff
@@ -4,30 +4,70 @@ Every aspect of flight dynamics is controlled by the selected "PID controller".
+keep itself stable. A craft with low P gain might feel as if it was "drunk". The pilot might feel as if they were constantly one step behind the quad--the pilot is responding to the quad, instead of the other way around.
+
+If P is set too high, the craft will rapidly oscillate/shake as it continually overshoots its target. Excess P gain is the single easiest thing to troubleshoot in multirotor tuning because the oscillations that result are so distinct. Excess P gain is demonstrated in [this video](https://www.youtube.com/watch?v=RMQXSB5-6bE). 
+
+A good guideline for setting P gain is to increase P until oscillations start to occur during normal control input (e.g. moderate turns and forward flight). Then decrease P to between 50% and 80% of that value. Another way of determining the oscillation threshold is to riase the throttle quickly from a hover and look/listen for oscillations during the punchout. Often times, espeically if flying line-of-sight, oscillations will be easier to hear than to see.
+
+If the multirotor is not perfectly symmetrical, the pitch and roll axes will need to have different gains. This can be mitigated to some degree by setting a [custom mix](Mixer.md). A proper custom mix will cause the P gains to be closer to the same on the pitch and roll axes. One approach is to set pitch and roll P gain to a value that is just barely high enough to be stable, then increase them one at a time until oscillations occur, to find the separate oscillation point of the two axes.
+
+Higher P gain tends to give a craft a sharper response to control inputs, but one of the most common problems made by novice tuners is having excess P. Even if oscillations aren't visible, there can be micro-oscillations that make their way to the motors, that can result in decreased flight time and hotter motors. In general, it's probably a good idea for a novice tuner to err slightly on the side of P being a bit low, rather than a bit high. Acrobatic flyers especially should use a P gain towards the lower end of the 50%-80% scale. More stable flyers, such as aerial camera operators, may use a higher P gain. This might seem counterintuitive, since acrobatic flyers would prefer to have a much sharper and more "locked in" feel than aerial camera operators, who benefit from a softer response. The reason for this is that high P gain is more likely to result in oscillations during extreme maneuvers and at high throttle. Therefore, pilots who will be making extreme maneuvers at high throttle must use a lower P gain than pilots who fly in a more docile style.
+
+**The I term**. Ideally, the P term would be all that is needed to have a perfect controller. In reality, the P term is always a little bit "behind the curve", because the copter takes some time to respond, and by the time it responds, the situation has changed. The I term tracks the cumulative error accrued by the P term and attempts to compensate for it. 
```

Comment:
The need for the I term is not because the P term is a bit late.  The P term being a bit delayed, in association with the momentum of the quad, is why we get overshoot wobble.  Dealing with overshoot wobble is why we have the D term.  The I term is not there because the P term is a bit late.

The I term is actually there because the P term must always be set slightly below the force required to fully correct an error.  If the P term resulted in a force that was was greater than the force required to completely correct an error, it would always 'overdo' the correction, causing uncontrollable oscillation.  

As a result, the P term alone will not fully correct for persistent offsets, especially small persistent offsets.  This is where the I term comes in.  It accumulates up these small errors over time and compensates for them.  

A good I analogy might be when a quadcopter in level mode but has a battery that has slipped a bit so the weigh distribution is now off-centre, such that it tends to persistently lean to one side.  A P response from the imbalance in weight will attempt to correct the lean, but cannot be quite strong enough - on its own - to entirely solve the problem.  The I term accumulates that error over time, and gradually builds enough force to entirely correct the lean and get the craft much closer to nearly perfectly level.  Likewise I is essential to deal with  slow variable external forces - like wind resistance at greater speed - which would otherwise to change the intended angle of the craft as we add throttle.